src/connectdlg.cpp

```diff
@@ -41,51 +41,63 @@ CConnectDlg::CConnectDlg ( CClientSettings* pNSetP, const bool bNewShowCompleteR
     setupUi ( this );
 
     // Add help text to controls -----------------------------------------------
-    // server list
+    // directory
+    QString strDirectoryWT = "<b>" + tr ( "Directory" ) + ":</b> " +
+                             tr ( "Selects the server list to be shown. "
+                                  "You can add custom directories in Advanced Settings." );
+    QString strDirectoryAN = tr ( "Directory combo box" );
+
+    lblList->setWhatsThis ( strDirectoryWT );
+    lblList->setAccessibleName ( strDirectoryAN );
+    cbxDirectoryServer->setWhatsThis ( strDirectoryWT );
+    cbxDirectoryServer->setAccessibleName ( strDirectoryAN );
+
+    // filter
+    QString strFilterWT = "<b>" + tr ( "Filter" ) + ":</b> " +
+                          tr ( "The server list is filtered by the given text. Note that the filter is case insensitive. "
+                               "A single # character will filter for those servers with at least one client connected." );
```

Comment:
Acc. [the Guide](https://jamulus.io/contribute/Style-and-Tone) I think we'd say "person" here not "client"

Reply:
It's musician elsewhere, too, which I dislike...  But a client connected just to stream into or out of Jamulus isn't a person, either...  I'll go with person, though.